File: Zeze/Util/Random.cs
using System.Collections.Generic;

namespace Zeze.Util
{
    public sealed class Random
    {
        public static System.Random Instance { get; } = new System.Random();

        public static IList<T> Shuffle<T>(IList<T> list)
        {
            for (int i = 1; i < list.Count; i++)
            {
                int pos = Instance.Next(i + 1);
                var x = list[i];
                list[i] = list[pos];
                list[pos] = x;
            }
            return list;
        }

        public static T[] Shuffle<T>(T[] list)
        {
            for (int i = 1; i < list.Length; i++)
            {
                int pos = Instance.Next(i + 1);
                var x = list[i];
                list[i] = list[pos];
                list[pos] = x;
            }
            return list;
        }
    }
}

File: ZezeJava/src/main/java/Zeze/Util/Random.java
package Zeze.Util;

import java.util.*;

public final class Random {
	private static final java.util.Random Instance = new java.util.Random();
	public static java.util.Random getInstance() {
		return Instance;
	}

	public static <T> List<T> Shuffle(List<T> list) {
		for (int i = 1; i < list.size(); i++) {
			int pos = getInstance().nextInt(i + 1);
			var x = list.get(i);
			list.set(i, list.get(pos));
			list.set(pos, x);
		}
		return list;
	}

	public static <T> T[] Shuffle(T[] list) {
		for (int i = 1; i < list.length; i++) {
			int pos = getInstance().nextInt(i + 1);
			var x = list[i];
			list[i] = list[pos];
			list[pos] = x;
		}
		return list;
	}

	public static void main(String []args) {
		System.out.println(Instance.nextLong());
	}
}